apps/wallet/src/ui/app/hooks/useAppResolveSuinsName.ts

```diff
@@ -0,0 +1,12 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+import { useFeatureIsOn } from '@growthbook/growthbook-react';
+import { normalizeSuiNSName } from '@mysten/sui.js/utils';
+
+import { useResolveSuiNSName } from '../../../../../core';
+
+export function useAppResolveSuinsName(address?: string) {
```

Comment:
`useAppResolveSuinsName` doesn't make a whole lot of sense to me - can we rename the import here and leave the hook name `useResolveSuinsName` ?

Reply:
Updated @mamos-mysten